samples/posix/gettimeofday/Makefile.host

```diff
@@ -1,4 +1,4 @@
 # This makefile builds the sample for a POSIX system, like Linux
 
-gettimeofday: src/main.c
-	$(CC) $^ -o $@
+gettimeofday: samples/posix/gettimeofday/src/main.c
+	$(CC) $^ -o samples/posix/gettimeofday/$@
```

Comment:
If we output the binary to a `build` folder, then it will be ignored by git and won't be an untracked file cc @kartben :

```suggestion
gettimeofday: src/main.c
	mkdir -p build
	$(CC) $^ -o build/$@
```

Reply:
Thanks for the suggestion. I made this change in my latest update.